plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs

```diff
@@ -140,6 +192,70 @@ private SplittedResult SplitModelToOverloadLevel(FileModel model)
             return new SplittedResult(primaryItem.Uid, children.OrderBy(s => GetDisplayName(s)), splittedModels);
         }
 
+        private string GetNewFileName(string filePath, MemberType? type, Dictionary<string, Tuple<bool, int>> existingFilePath)
+        {
+            string newFileName;
+            if (existingFilePath.TryGetValue(filePath, out Tuple<bool, int> nameStatus))
+            {
+                existingFilePath[filePath] = Tuple.Create(true, nameStatus.Item2);
```

Comment:
>Tuple.Create(true, nameStatus.Item2); [](start = 45, length = 37)

You can use a private sealed class instead of tuple, for better readability, and you can also assign `true` here directly.